packages/@react-spectrum/list/src/ListViewItem.tsx

```diff
@@ -62,8 +90,50 @@ export function ListViewItem(props) {
         }
         ref={ref}
         {...mergedProps}>
-        {item.rendered}
+        <Grid UNSAFE_className={listStyles['react-spectrum-ListViewItem-grid']} UNSAFE_style={{minHeight: '32px'}}>
+          {showCheckbox &&
+          <Checkbox UNSAFE_style={{marginLeft: '4px', paddingRight: '0'}} {...checkboxProps} isEmphasized />}
+          <SlotProvider
+            slots={{
+              content: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-content']},
+              text: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-content']},
+              description: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-description']},
+              icon: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-icon'], size: 'M'},
+              image: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-image']},
+              actionGroup: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-actions'], isQuiet: true, density: 'compact'},
+              actionButton: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-actions'], isQuiet: true},
+              actionMenu: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-actions'], isQuiet: true},
+              link: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-content'], isQuiet: true}
+            }}>
+            {typeof item.rendered === 'string' ? <Content>{item.rendered}</Content> : item.rendered}
+            {chevron}
+          </SlotProvider>
+        </Grid>
       </div>
     </div>
   );
 }
+
+function useListSelectionCheckbox(props, state) {
```

Comment:
Should be in aria package? Should we move the table one into grid if it's the same?